docs/whats-new/toc.yml

```diff
@@ -1,20 +1,18 @@
-- name: What's new
-  href: index.yml
-  expanded: true
-  items:
-    - name: 2020
-      items:
-      - name: May
-        href: dotnet-2020-05.md
-      - name: April
-        href: dotnet-2020-04.md
-      - name: March
-        href: dotnet-2020-03.md
-      - name: February
-        href: dotnet-2020-02.md
-      - name: January
-        href: dotnet-2020-01.md
-    - name: 2019
-      items:
-      - name: December
-        href: 2019-12.md
```

Comment:
Also, should the PR actually **remove** the *2019-12.md* file?

Reply:
It does not at the moment. It's a decision we have to make in the future if we want to keep the links active or not.

Reply:
> It does not at the moment. It's a decision we have to make in the future if we want to keep the links active or not.

It's actually deleted currently.

Reply:
We should remove the files when they are removed from the TOC. It's a known issue that it will 404, but the traffic after 6 months is really low.